monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy

```diff
@@ -11,7 +11,11 @@ class MonkeyConfigurationPlugin implements Plugin<Project> {
     @Override
     void apply(Project project) {
         ensureAndroidPluginAppliedTo(project)
-        ensureCommandPluginAppliedTo(project)
+
+        project.configure(project) {
```

Comment:
Can't we just do
`project.apply plugin: 'com.novoda.android-command'`

`apply` is a method in `Project` class.